helm/templates/configmap.yaml

```diff
@@ -7,3 +7,6 @@ data:
   config.yaml: |
     listen_address: 0.0.0.0:{{ .Values.app.containerPort }}
     developer_mode: true
+    fleetdb:
+      endpoint: http://fleetdb:8000
```

Comment:
This should probably be in `values.yaml`

Reply:
Updated.